testdata/time.go

```diff
@@ -11,3 +11,27 @@ import (
 var rpcTimeoutMsec = flag.Duration("rpc_timeout", 100*time.Millisecond, "some flag") // MATCH /Msec.*\*time.Duration/
 
 var timeoutSecs = 5 * time.Second // MATCH /Secs.*time.Duration/
+
+var delay = flag.Duration("delay", 5, "wait impatiently") // MATCH /time\.Duration without unit: 5/
+
+func f(...time.Duration) {}
+
+var (
+	_ = time.Duration(15)
```

Comment:
Okay, the message probably isn't great for `time.Duration(30) * time.Second` either, even though we'd typically ask people to change it. I'm fine with not catching this (at least right now), but please add a comment to the test data file saying why.